rtos/TARGET_CORTEX/SysTimer.cpp

```diff
@@ -37,6 +37,9 @@ extern "C" {
 #endif
 }
 
+#define US_IN_TICK          (1000000 / (OS_TICK_FREQ))
+MBED_STATIC_ASSERT(1000000 == US_IN_TICK * OS_TICK_FREQ, "OS_TICK_FREQ must be a divisor of 1000000 for correct tick calculations");
```

Comment:
I'd have phrased that `1000000 % OS_TICK_FREQ == 0`, but whatever.

However, there's an inconsistency here - you shouldn't need the `()` around OS_TICK_FREQ above, but if you do, you'd need them here too.

Reply:
done